<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWidget</class>
 <widget class="QWidget" name="PlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Plot</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0">
       <widget class="QLabel" name="zoomLabel">
        <property name="text">
         <string>Default zoom</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="gridVerticalSpaceLabel">
        <property name="text">
         <string>Grid vertical spacing</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="imgGeneratorBufferLabel">
        <property name="text">
         <string>Image generator buffer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="frameWidthSpinBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="gridHorizontalSpaceLabel">
        <property name="text">
         <string>Grid horizontal spacing</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="gridVerticalSpinBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="frameWidthSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="frameWidthLabel">
        <property name="text">
         <string>Frame width</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="gridHorizontalSpinBox"/>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="imgGeneratorBufferSpinBox"/>
      </item>
      <item row="3" column="2">
       <widget class="QSlider" name="gridHorizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="gridVerticalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSlider" name="imgGeneratorBufferSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSlider" name="zoomSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSpinBox" name="zoomSpinBox">
        <property name="suffix">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="gridVisibilityCheckBox">
        <property name="text">
         <string>Show Grind</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
